src/EchoToImageLUT.hpp

```diff
@@ -44,15 +42,31 @@ namespace radar_base {
         EchoToImageLUT(int num_angles, int sweep_size, float beam_width, int window_size);
         ~EchoToImageLUT() = default;
 
-        void updateImage(cv::Mat& image,
-            int angle,
-            int echo_index,
-            int echo,
-            bool force_write = false) const;
+        void updateImage(cv::Mat& image, int angle, int echo_index, int echo) const;
+
+        /**
+         * Checks if provided configuration matches the LUT.
+         *
+         * @param num_angles Amount of samples contained in a radar revolution.
+         * @param sweep_size Number of measurements inside a single sweep.
+         * @param beam_width Angle between the 3dB half power points on the main lobe.
+         * @param window_size The param will create a squared frame.
+         * @return true if the LUT configuration matches
+         * @return false if the LUT configuration doesn't match
+         */
+        bool hasMatchingConfiguration(int num_angles,
+            int sweep_size,
+            float beam_width,
+            int window_size);
 
         void linearizeRawTable(RawTable const& table);
 
         cv::Point fetchEntry(int angle, int echo_index) const;
+
+        int const lut_sweep_size;
+        int const lut_num_angles;
+        float const lut_beam_width;
+        int const lut_window_size;
```

Comment:
Do they really need the lut prefix? We are already inside the LUT context

Reply:
The point of using lut_ prefix was to not have ambiguous naming with the function variables, however, as @jhonasiv pointed out, it is better to add a sufix to the variables(such as "_") instead.